crypto: update to nrf_oberon 3.0.15 (#1317)
nrfconnect/sdk-nrfxlib: crypto/doc/CHANGELOG_oberon.rst

All notable changes to this project are documented in this file.

nrf_oberon - 3.0.15
*******************

New version of the nrf_oberon library with the following changes.

Added
=====

> * Added missing ocrypto functions for SECP160 curves which broke some configurations.

Reviewer: The grammatical fix here depends on the intended meaning.

There should always be a comma after the word before `which`, which in this case would read, "SECP160  curves, which broke some configurations." 

However, I read this construction as meaning that adding the missing functions broke some configurations.

If this is not the intended meaning, use:

"SECP160 curves that broke..."

I read this as: missing functions broke some configurations, and adding them fixes it.

Author: I was intended to write that the fact that we had the missing functions broke some configurations. Now that I added them back they should be fixed. So I updated the text based on your second suggestion.

> * Fixed a bug which resulted in AEAD encryption/decryption with AES-CCM producing incorrect results for large buffers (>1MB data with 12 byte nonce).

Reviewer: ```suggestion
* Fixed a bug that caused AEAD encryption/decryption with AES-CCM to produce incorrect results for large buffers (>1MB data with 12 byte nonce).
```
